musicbot/bot.py

```diff
@@ -75,11 +76,13 @@ def __init__(self, config_file=None, perms_file=None):
 
         self.blacklist = set(load_file(self.config.blacklist_file))
         self.autoplaylist = load_file(self.config.auto_playlist_file)
+        self.aliases = json.load(open(self.config.command_aliases_file))
```

Comment:
This isn't really good enough. This will stop execution if it throws an exception, such as a `JSONDecodeError` (for example, when the file isn't valid JSON). There needs to be additional checks for this. It may also be better to handle it in `config.py` instead and sanity check it there.

Reply:
Moved aliases setup to config.py and added checks there